src/main/kotlin/ch/wisv/choice/document/service/DocumentServiceImpl.kt

```diff
@@ -72,9 +71,8 @@ class DocumentServiceImpl(val documentRepository: DocumentRepository,
 
     override fun deleteDocumentByExamId(id: Long) {
         val document = getDocumentByExamId(id)
-        if (document.file != null) {
-            fileRepository.delete(document.file)
-        }
+        if(document.file != null)
+            fileRepository.delete(document.file!!)
```

Comment:
Zelfde hier, is de `!!` nodig?

Reply:
Genereert compile error wanneer de `!!` niet aanwezig is